main.go

```diff
@@ -5,29 +5,85 @@
 package main
 
 import (
+	"bytes"
+	"errors"
+	"github.com/Azure/azure-sdk-for-go/storage"
 	"html/template"
 	"io/ioutil"
 	"net/http"
+	"os"
 	"regexp"
 )
 
+var blobService storage.BlobStorageClient
+var containerName = os.Getenv("AZURE_STORAGE_CONTAINER_NAME")
+
 type Page struct {
 	Title string
 	Body  []byte
 }
 
+func initAzureStorage() {
+	var storageKey = os.Getenv("AZURE_STORAGE_ACCESS_KEY")
+	var storageName = os.Getenv("AZURE_STORAGE_ACCOUNT")
+	client, err := storage.NewBasicClient(storageName, storageKey)
+	if err != nil {
+		panic(err)
```

Comment:
Could we return an error from this function instead of panicing, and handle the error in `main` by writing to stderr?